jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/grid/DataGrid.java

```diff
@@ -59,6 +59,7 @@
 
 public class DataGrid<E> extends JmixGrid<E> implements ListDataComponent<E>, MultiSelectLookupComponent<E>,
         EnhancedDataGrid<E>, SupportsEnterPress<DataGrid<E>>, ApplicationContextAware, InitializingBean {
+    public static final String QUALIFIED_XML_NAME = "dataGrid";
```

Comment:
What is the purpose of the constant? Is it used somewhere else? I'm not sure that we should introduce a constant here.

1. It seems that constant name is not correct. Is this a **qualified** name? In my understanding a qualified name is the name that consist of a schema name and a local name, e.g. `xsl:template`. I don't see any schema here.
2. If we start adding a constant with element name to classes then it looks like a pattern. It is expected that each element must have its name in a constant. But we don't need this yet and also it's not always possible (because some components come from Vaadin and we don't have classes for them inside Jmix.